.gitignore

```diff
@@ -68,6 +68,7 @@ environment_log.txt
 #################
 Documentation/html_viewer/resources/data/api/*.js
 Documentation/html_viewer/resources/data/guide/*.js
+Documentation/html_viewer/resources/data/
```

Comment:
@mwieder how about changing the above lines to a catch all `Documentation/html_viewer/resources/data/*/*.js`